src/main/java/core/basesyntax/service/FileReaderService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.io.File;
+import java.util.List;
+
+public interface FileReaderService {
+    List<String> fileToStringList(File file);
```

Comment:
I'd recommend renaming this method and take a filepath as an argument instead of file